app/facade/food_facade.rb

```diff
@@ -0,0 +1,9 @@
+class FoodFacade
+  def self.ingredient_search(search_terms)
+    food_results = FoodService.food_search(search_terms)
+    food_array = food_results[:foods].map do |food|
+      Food.new(food)
+    end
+    [food_results[:totalHits], food_array[0..9]]
```

Comment:
ooh, nice work here! I like this syntax a lot, that way you don't need to make the service call again for the totalHits.

Reply:
I believe there's also a way to limit the number of food responses you get back from the array, I'm curious if that would affect your total hits though. I believe to limit the responses, you can add a parameter on the api call.